### DISCUSSION TITLE
Image Path

### DISCUSSION THREAD

**Discussion by Blanchdog** [rdeepak2002/reddit-place-script-2022#360] [Q&A] [answered]
I'm trying to get this working on windows, but when I put it into the json linter it doesn't like the image path line. I think it has something to do with the backslashes, since when I just put the file name in instead of the file path the linter is fine with it. What's the correct thing to do here? And once I have the json all set, where is the config.json I'm supposed to move it too? Does it mean I need to rename the json from config_example.json to just config.json?

**Comment by Skyline-9** [comment, marked as answer]
> What's the correct thing to do here?

Try leaving the image in the root directory, like how image.jpg is

> And once I have the json all set, where is the config.json I'm supposed to move it too? Does it mean I need to rename the json from config_example.json to just config.json?

Yes, once you're done configuring the json rename it to config.json

**Reply by Blanchdog** [reply]
Awesome thank you. One more question, how do I get it to run? I've tried running it from cmd and then running the path, but after going through a few steps it exits out with an error. It's too fast for me to read, but it's something about pip dependencies? Normally I use anaconda, but I also installed python through the microsoft store and ran the document to install pip. Just can't seem to get it to work.

**Reply by Skyline-9** [reply]
Can you post some detailed logs?

**Reply by Blanchdog** [reply]
I would, but I'm not sure how to keep them open long enough to even read them, let alone post them. The terminal just closes as soon as it's done and I'm not sure how to stop it.

**Reply by Skyline-9** [reply]
Can you try manually running `pip install -r requirements.txt`?

**Comment by thoroc** [comment]
It could be simple to add a path to the image as argument to the cli and set that instead of the config. This would allow for image stored somewhere else on the computer. Probably use `pathlib` as well to handle cross OS paths in a OOP manner.
